package.json

```diff
@@ -26,6 +26,7 @@
         "@typescript-eslint/eslint-plugin": "^6.7.4",
         "eslint": "^8.19.0",
         "eslint-plugin-jest": "^27.4.2",
+        "hls.js": "^1.5.18",
```

Comment:
@KrzysztofMoch You mean `peerDependencies`?

Reply:
@KrzysztofMoch like this?

```
"peerDependencies": {
        "react": "*",
        "react-native": "*",
        "hls.js": "^1.5.18"
 },
```

Reply:
@bhyoo99 The best would be to put it in optionnalDependency: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies
It will allow to not embed the package is not needed.
It would be interesting to put the information in the documentation also.